live_meetings.md

```diff
@@ -0,0 +1,23 @@
+With thanks to the Portland Python User Group - https://www.meetup.co...
+
+## Harassment
+
+The Pittsburgh AWS User Group is dedicated to providing a harassment-free community for everyone, regardless of sex, gender identity or expression, sexual orientation, disability, physical appearance, age, body size, race, nationality, or religious beliefs.
+
+Harassment includes offensive verbal or written comments related to sex, gender identity or expression, sexual orientation, disability, physical appearance, age, body size, race, nationality, or religious beliefs, deliberate intimidation, threats, stalking, doxing, brigading, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention. Sexual language and imagery is not appropriate for any Pittsburgh AWS User Group event or communication channel. Community members asked to stop any harassing behavior are expected to comply immediately.
+
+## Self Promotion & Soliciation
+
+Community members should not use the meetup, or meetup resources for promotion or solicitationwithout approval from the organizers. This includes but is not limited to blogs, social media, external websites, video sharing sites.
```

Comment:
```suggestion
Community members should not use the user group, or user group resources for promotion or solicitation without approval from the organizers. This includes but is not limited to blogs, social media, external websites, video sharing sites.
```

Only the spacing issue is needed here, the use of meetup versus user group is up for discussion, but I don't like playing into their brand.